src/pages/home/report/ReportActionItem.js

```diff
@@ -1,28 +1,161 @@
+     */
+    isInReportActionContextMenuBeta() {
+        return _.contains(this.props.betas, CONST.BETAS.REPORT_ACTION_CONTEXT_MENU)
+            || _.contains(this.props.betas, CONST.BETAS.ALL);
+    }
+
+    /**
+     * Save the location of a native press event.
+     *
+     * @param {Object} nativeEvent
+     */
+    capturePressLocation(nativeEvent) {
+        this.popoverAnchorPosition = {
+            horizontal: nativeEvent.pageX,
+            vertical: nativeEvent.pageY,
+        };
+    }
+
+    /**
+     * Show the ReportActionContextMenu modal popover.
+     *
+     * @param {Object} [event] - A press event.
+     */
+    showPopover(event) {
+        const nativeEvent = event.nativeEvent || {};
+        this.capturePressLocation(nativeEvent);
+        if (this.isInReportActionContextMenuBeta()) {
+            this.setState({isPopoverVisible: true});
+        }
+    }
+
+    /**
+     * Hide the ReportActionContextMenu modal popover.
+     */
+    hidePopover() {
+        this.setState({isPopoverVisible: false});
+    }
+
+    render() {
+        return (
+            <PressableWithSecondaryInteraction onSecondaryInteraction={this.showPopover}>
+                <Hoverable>
+                    {hovered => (
+                        <View>
+                            <View style={getReportActionItemStyles(hovered)}>
+                                {!this.props.displayAsGroup
+                                    ? <ReportActionItemSingle action={this.props.action} />
+                                    : <ReportActionItemGrouped action={this.props.action} />}
+                            </View>
+                            <View style={styles.miniReportActionContextMenuWrapperStyle}>
+                                <ReportActionContextMenu
+                                    reportID={this.props.reportID}
+                                    reportActionID={this.props.action.sequenceNumber}
+                                    isVisible={
+                                        hovered
+                                        && this.isInReportActionContextMenuBeta()
+                                        && !this.state.isPopoverVisible
+                                    }
+                                    isMini
+                                />
+                            </View>
+                            <PopoverWithMeasuredContent
+                                isVisible={this.state.isPopoverVisible}
+                                onClose={this.hidePopover}
+                                anchorPosition={this.popoverAnchorPosition}
+                                animationIn="fadeIn"
+                                measureContent={() => (
+                                    <ReportActionContextMenu
+                                        isVisible
+                                        reportID={-1}
+                                        reportActionID={-1}
+                                    />
```

Comment:
> Why are we not just providing a constant width / height?

Mostly just because we want the `ReportActionContextMenu` to be responsively sized, and adding or removing elements from the menu is as simple as adding or removing items from the `CONTEXT_ACTIONS` array.

_However_, one potential simplification would be to eliminate the `measureContent` component and just measure the children (and pass a `ReportActionContextMenu` that's always visible in the children).